intro_to_ada.rst

```diff
@@ -530,28 +520,21 @@ more details.
 Strongly typed language
 =======================
 
-Ada is a seriously typed language. It is interestingly modern in that
+Ada is a strongly typed language. It is interestingly modern in that
 aspect: Strong static typing is going through a popularity rise, due to
 multiple factors: Popularity of statically typed functional programming,
 a big push from the academic community in the typing domain, many
 practical languages with strong type systems emerging, etc.
 
-.. gusthoff: "seriously typed"? Seriously?! ;-)
```

Comment:
Good. I liked that you kept the first part but removed the "Hipster language" part ;)